contributing/getting-started-quick.md

```diff
@@ -25,11 +25,11 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
 
 * [Install and set up Git](https://help.github.com/articles/set-up-git/); in the "Authenticating" step of that page use SSH instead of HTTPS
 
-* Install [NodeJS](https://nodejs.org/)
+* Install [NodeJS](https://nodejs.org/) version >= 4.7
 
-* Install [yarn](https://yarnpkg.com/en/docs/install)
+* If the version of [npm](https://www.npmjs.com/)` that was installed with NodeJS is lower than version 5, upgrade it by running `npm install -g npm@latest` (on some platforms this command might require elevated privileges using `sudo`)
```

Comment:
This line is badly formatted due to a stray backtick. (You can click the "view" button above this file in the "files changed" view to see what it will actually look like.)

Reply:
Done